### ISSUE TITLE
[gatsby-regype-inline-images]: macOS build error 'dyld: lazy symbol binding failed: Symbol not found: _g_once_i'

### ISSUE THREAD

**Issue by styxlab** [styxlab/gatsby-theme-try-ghost#190]
```
$ gatsby develop
success open and validate gatsby-configs - 0.865s
⠋ load plugins
dyld: lazy symbol binding failed: Symbol not found: _g_once_impl
  Referenced from: /Users/andy_lap/Documents/github/pascalandy/jamify-starter/node_modules/gatsby-rehype-inline-images/node_modules/sharp/build/Release/sharp.node
  Expected in: /Users/andy_lap/Documents/github/pascalandy/jamify-starter/node_modules/sharp/build/Release/../../vendor/lib/libvips.42.dylib

dyld: Symbol not found: _g_once_impl
  Referenced from: /Users/andy_lap/Documents/github/pascalandy/jamify-starter/node_modules/gatsby-rehype-inline-images/node_modules/sharp/build/Release/sharp.node
  Expected in: /Users/andy_lap/Documents/github/pascalandy/jamify-starter/node_modules/sharp/build/Release/../../vendor/lib/libvips.42.dylib

error Command failed with signal "SIGKILL".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by styxlab** [comment]
#### System specs of test system
1. macOS High Sierra 10.13.2 or higher
2. `node -v`: v14.13.0
3. `yarn -v`: 1.22.5

#### Steps to reproduce issue
```bash
git clone https://github.com/styxlab/gatsby-starter-try-ghost.git jamify-starter
cd jamify-starter
yarn add gatsby-rehype-inline-images
yarn
```
Add `gatsby-rehype-inline-images` to `gatsby-config.js`
```javascript
// gatsby-config.js

plugins: [
    
    {
        resolve: `gatsby-transformer-rehype`,
        options: {
            filter: node => (
                node.internal.type === `GhostPost` ||
                node.internal.type === `GhostPage`
            ),
            plugins: [
                {
                    resolve: `gatsby-rehype-ghost-links`,
                },
                {
                    resolve: `gatsby-rehype-prismjs`,
                },
                {
                    resolve: `gatsby-rehype-inline-images`,
                },
            ],
        },
    },   
]
```
Start building...
```
yarn
yarn build
```

**Comment by styxlab** [comment]
Fixed by downgrading `sharp` to version `0.25.4` in `gatsby-rehype-inline-images`. Updated all packages on npm. If you run into this issue, make sure you have the following version installed: `gatsby-rehype-inline-images`: >=1.7.2.
